spring-kafka/src/main/java/org/springframework/kafka/core/reactive/ReactiveKafkaProducerTemplate.java

```diff
@@ -71,6 +72,13 @@ public ReactiveKafkaProducerTemplate(SenderOptions<K, V> senderOptions, RecordMe
 		this.messageConverter = messageConverter;
 	}
 
+	public ReactiveKafkaProducerTemplate(KafkaSender<K, V> sender, RecordMessageConverter messageConverter) {
```

Comment:
We should probably also add a variant that does not take a converter (to be consistent with the other ctors).